gwadoc/doc_en.py

```diff
@@ -1152,7 +1147,7 @@
 ### Relation: subevent EDP35
 
 relations.subevent.name.en = "Subevent"
-relations.subevent.df.en = "Concept B takes place during or as part of concept A, and whenever concept B takes place, concept A takes place"
+relations.subevent.df.en = "a concept for which a given concept is a part of"
```

Comment:
This definition is much simplified from the original and seems to lose the "takes place during" aspect (unlike the subevent-of relation below). I don't recall the motivation for subevents, so I just want to bring some attention here in case there was a mistake.